Week1/homework/js-exercises/logString.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+let myString = "Stefanos Leventis";
+console.log("My full name");
+console.log(myString);
+myString = "The Arkitekt";
+console.log("My nickname as a music producer");
+console.log(myString);
```

Comment:
In line 2, the value of the variable `myString` is `"Stefanos Leventis"`, which happens to be your full name! So, if we want to be 100% precise, the line 3 instead, should be `console.log("Stefanos Leventis");`

In the line 6, same as above, the new value of the `myString` is `"The Arkitekt"`, which again, happens to be your nickname as a music producer...

 ;-D     ...yeah yeah I know....